commit 6bd6343e22968878c35936581096a4cb6fb2b65b
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Wed Feb 21 15:36:10 2018 +0000

    ARM: dts: bcm2710 - local_intc users need polarity
    
    Someone upstream decided it was vital that interrupt signals internal
    to the BCM283x CPUs had a defined polarity, even though we don't know
    what it is. As a result the local_intc node now demands two-cell
    declarations and all users have to change to match or risk a warning
    from the DTC (at least).
    
    For what it's worth, LEVEL_HIGH was chosen.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm2710.dtsi b/arch/arm/boot/dts/bcm2710.dtsi
index 6fb5ed655d63..56d58f588203 100644
--- a/arch/arm/boot/dts/bcm2710.dtsi
+++ b/arch/arm/boot/dts/bcm2710.dtsi
@@ -14,7 +14,7 @@
 			compatible = "arm,cortex-a7-pmu";
 #endif
 			interrupt-parent = <&local_intc>;
-			interrupts = <9>;
+			interrupts = <9 IRQ_TYPE_LEVEL_HIGH>;
 		};
 
 		/delete-node/ timer@7e003000;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1147_6bd6343e22968878c35936581096a4cb6fb2b65b.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1147_6bd6343e22968878c35936581096a4cb6fb2b65b.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
